XcodeServerSDK/Server Entities/DeviceSpecification.swift

```diff
@@ -19,12 +19,14 @@ public class DevicePlatform : XcodeServerEntity {
         case iOS_Simulator = "com.apple.platform.iphonesimulator"
         case OSX = "com.apple.platform.macosx"
         case watchOS = "com.apple.platform.watchos"
-        case watchOS_Simulator = "com.apple.platform.watchsimulator"
+        case watchOS_Simulator = "com.apple.platform.watchsimulator" // Why is it here?
```

Comment:
Oh I see, you do have simulator type below. Can you also add the `TV` as `tvOS_Simulator` to this enum as well?